e2e/mv_test.go

```diff
@@ -311,3 +311,51 @@ func TestMoveMultipleS3ObjectsToS3(t *testing.T) {
 		assert.Assert(t, ensureS3Object(s3client, bucket, "dst/"+filename, content))
 	}
 }
+
+// mv s3://bucket/single*.blob* s3://bucket/single/
+func TestMoveWildcardTwoThousandS3ObjectsToS3(t *testing.T) {
```

Comment:
This test does not fail on `master` branch. I was expecting a timeout when running this on master.
 Please double check if we are testing the right assumptions.

Reply:
hmm, the example is in some sense close imitation of the one from [the related issue](https://github.com/peak/s5cmd/issues/168), but to test it I think any number of files above 1000 would do, so let me try some lower number than 2000.